app/views/manifests/show.json.rabl

```diff
@@ -0,0 +1,9 @@
+object false
+
+node(:name)        { Setting.site_title }
+node(:description) { strip_tags(Setting.site_description.presence || I18n.t('about.about_mastodon')) }
+node(:icons)       { [{ src: '/android-chrome-192x192.png', sizes: '192x192', type: 'image/png' }] }
+node(:theme_color) { '#282c37' }
+node(:display)     { 'standalone' }
```

Comment:
In the future it'd be nice if this were customizable on a per-user basis. Some toot.cafe users have said they would like to keep the URL bar when launched in "app" mode